src/systems/collision.rs

```diff
@@ -161,6 +161,91 @@ impl<'s> System<'s> for PincerCollisionSystem {
     }
 }
 
+pub struct FlierCollisionSystem;
```

Comment:
Marine's collision system, in this same file, also needs to be updated to account for the collision with the Flier. The `is_collidable` flag needs to be set to `false` on collision with the Flier, same way it is done for the collision with the Pincer. This will also take care switching Marine's animation to the `Die`.

Reply:
Done 👍